devops/sql/migrate-22.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE chatroach.surveys_metadata (
+	surveyid UUID NOT NULL REFERENCES chatroach.surveys(id) ON DELETE CASCADE,
```

Comment:
OK - this is where our naming convention problems come in. It shouldn't be surveyid (which is really a "form"), it should be shortcode.

Reply:
@calufa - any movement on this? This is a problem, because the off time shouldn't be unique to each "survey", which is really a "version of a survey", correct?

Reply:
In my mind, off time applies to a shortcode and pageid. Pageid gets you to survey user. And shortcode + survey user should be unique per metadata.

Reply:
@calufa - just following up on this! Any thoughts?